data-sources/us-tiger/convert.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+set +x
+
+INPATH="/tmp/tiger2018raw"
+INREGEX='_([0-9]{5})_edges.zip'
+NUM_EXPECTED=3233
+
+OUTPATH="/tmp/tiger2018parsed"
+mkdir -p "$OUTPATH"
+
+WORKPATH="/tmp/tiger2018work"
+mkdir -p "$WORKPATH"
+
+
+
+INFILES=($INPATH/*.zip)
+if [[ ${#INFILES[*]} != $NUM_EXPECTED ]]; then
+    echo "Expected $NUM_EXPECTED files. Found ${#INFILES[*]}."
+    exit 1
+fi
```

Comment:
I don't think that's a good idea because a) there is no guarantee that the number does not change over the years and then the script becomes incompatible and b) sometimes one only wants to convert a couple of files for testing.